### ISSUE TITLE
recording app: request date input change for new archival section

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#3014]
The archival input is very helpful already. I run into two places where I get slowed down significantly, at least one of which I hope you can do something about.

The date input format for some reason needs me to choose from a calendar, I cannot type in a date as I sometimes can elsewhere. These dates are always not the present, usually many years or decades ago, and scrolling through the calendar to get to the right date is the thing that is taking me longest in inputting real records.

The other thing is the OPs. It reallllly would be helpful to be able to create OPs in the OP link section, same way we can create loci in the relations. I am usually creating a record for stuff that has not been put in yet. So the OPs usually don’t exist. I have to exit, create the OPs, then go back and link them. This is a step I find I often skip because it is cumbersome, but that’s some of the most important information.

**Comment by urapadmin** [comment]
- well, let's make that two requests and move the OP thing to another ticket: #3015

**Comment by urapadmin** [comment]
what dates are we talking about here? I don't see a calendar anywhere. Are you on an iPad? 
I have not used a calendar control in FileMaker so this must come from somewhere else. Screenshot please.

**Comment by lbestock** [comment]
The starts/ends dates. I am doing this on an iPad indeed. Right. Argh. It is SO much easier to do this on the iPad pro with the keyboard attached - I get all the advantages of a computer and all the advantages of an iPad rolled into one. Easy typing, easy photos.

**Comment by urapadmin** [comment]
that still does not explain why there are date pickers where there should not be. So please show me what that looks like. Are you sure you can't type anything in there? I can type our usual latin dates.

**Comment by lbestock** [comment]
I really can’t type anything. 
![IMG_0083](https://github.com/user-attachments/assets/9cf1c5e5-e3bf-49be-862b-d232d8ae185b)

**Comment by urapadmin** [comment]
try that: 
![image](https://github.com/user-attachments/assets/7ae75000-b8f5-403b-b1dd-f41ec7f9ab7e)

**Comment by urapadmin** [comment]
also: this must be the case absolutely everywhere, no?

**Comment by lbestock** [comment]
I don’t know if it is everywhere. I just got this iPad and have never tried using an iPad for old data before because they are such a relative pain to type on. Let me check.

**Comment by lbestock** [comment]
Yup wow. It is everywhere. And the little keyboard button is the answer. And now this will be great.

**Comment by urapadmin** [comment]
I looked it up: It is an issue that users have reported to Claris since version 19, complaining that a date picker is often enough not a good way to enter a date (really not news). We are at 21 now and can safely say that whatever Claris cooks up, we have to eat it.

**Comment by urapadmin** [comment]
bit surprising that it has not come up before. So I have the feeling there is still something up here ...

**Comment by urapadmin** [comment]
### recording app 16.7.2
- I reopen this because I have tried something with the starts / ends input fields. Does the stupid calendar still pop up? I am trying to enforce an ASCII keyboard here.

**Comment by lbestock** [comment]
Alas, the stupid calendar still pops up.

**Comment by urapadmin** [comment]
I need to experiment with this myself but I fear it is one of those splendid Apple or Claris features nobody asked for.

**Comment by urapadmin** [comment]
there seems to be nothing I can do here.
